README.md

```diff
@@ -39,6 +47,51 @@ py_binary(
 )
 ```
 
+## Importing `pip` dependencies
+
+These rules are designed to have developers continue using `requirements.txt`
+to express their dependencies in a Python idiomatic manner.  These dependencies
+are imported into the Bazel dependency graph via a two-phased process in
+`WORKSPACE`:
+
+```python
+load("@io_bazel_rules_python//python:pip.bzl", "pip_import")
+
+# This rule translates the specified requirements.txt into
+# @my_deps//:requirements.bzl, which itself exposes a pip_install method.
+pip_import(
+   name = "my_deps",
+   requirements = "//path/to:requirements.txt",
+)
+
+# Load the pip_install symbol for my_deps, and create the dependencies'
+# repositories.
+load("@my_deps//:requirements.bzl", "pip_install")
+pip_install()
+```
+
+## Consuming `pip` dependencies
+
+Once a set of dependencies has been imported via `pip_import` and `pip_install`
+we can start consuming them in our `py_{binary,library,test}` rules.  In support
+of this, the generated `requirements.bzl` also contains a `packages` method,
+which can be used directly in `deps=[]` to reference an imported `py_library`.
+
+```python
+load("@my_deps//:requirements.txt", "packages")
+
+py_library(
+    name = "mylib",
+    srcs = ["mylib.py"],
+    deps = [
+        ":myotherlib",
+	# This takes the name as specified in requirements.txt
+	packages("importeddep"),
```

Comment:
Renamed.  This is a vestige of a time when I was exporting `packages` as a dict of packages, but I had to make it a function to allow passing in `_` instead of `-`.